packages/forms/frontend/sirius-components-forms/src/views/DiagramFilterView.tsx

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { WorkbenchViewComponentProps } from '@eclipse-sirius/sirius-components-core';
```

Comment:
They are required by `FormBasedView`, I used my own props through the extension mechanism (see `DiagramPanelActionProps` in `DiagramPanel.types`), but in this particular file (which is now renamed `DiagramFilterForm`) I still need this `WorkbenchViewComponentProps` because it is required by the `FormBasedView`. 

I am maybe missing something, but I don't see how to get rid of it.